hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/RowResourceBase.java

```diff
@@ -18,22 +18,16 @@
 package org.apache.hadoop.hbase.rest;
 
 import static org.junit.Assert.assertEquals;
-
 import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
We aren't shading jackson-databind? Shouldn't we, if we are shading other jackson packages? jackson.databind has been a source of several CVEs so is definitely something we want to control and keep an eye on.

Reply:
See above -- it seems we do not currently shade jackson-databind and the patch as posted does not do so either.